TRANSLATING.md

```diff
@@ -195,10 +196,9 @@ Translators are free to create and mantain a glossary, style guide and other tip
 
 Feel free to take a look at the [Deutsch Guide](https://github.com/withastro/docs/blob/main/src/i18n/de/README.md) for an example.
 
-
 ### Frontmatter
 
-Our pages are generated from Markdown files which have frontmatter properties. These are variables that hold information about the page (values) that we later use to specify the page's title, description, and other special data. 
+Our pages are generated from Markdown files which have frontmatter properties. These are variables that hold information about the page (values) that we later use to specify the page's title, description, and other special data.
```

Comment:
I think a peep looking to translate our docs will already know what MDX is, so I don't think we need to explain it, I'd just change to MDX, IMO. Feel free to convince me otherwise!
```suggestion
Our pages are generated from MDX files which have frontmatter properties. These are variables that hold information about the page (values) that we later use to specify the page's title, description, and other special data.
```